cluster/examples/kubernetes/ceph/filesystem-mirror.yaml

```diff
@@ -0,0 +1,38 @@
+#################################################################################################################
+# Create filesystem-mirror daemon
+#  kubectl create -f filesystem-mirror.yaml
+#################################################################################################################
+
+apiVersion: ceph.rook.io/v1
+kind: CephFilesystemMirror
+metadata:
+  name: my-fs-mirror
+  namespace: rook-ceph # namespace:cluster
+spec:
+  # The affinity rules to apply to the mds deployment
+  placement:
+  #  nodeAffinity:
+  #    requiredDuringSchedulingIgnoredDuringExecution:
+  #      nodeSelectorTerms:
+  #      - matchExpressions:
+  #        - key: role
+  #          operator: In
+  #          values:
+  #          - rgw-node
+  #  tolerations:
+  #  - key: cephfs-mirror-node
+  #    operator: Exists
+  #  podAffinity:
+  #  podAntiAffinity:
+  # A key/value list of annotations
+  annotations:
+  #  key: value
+  resources:
+  # The requests and limits, for example to allow the cephfs-mirror pod(s) to use half of one CPU core and 1 gigabyte of memory
+  #  limits:
+  #    cpu: "500m"
```

Comment:
Any idea on recommended limits? Or these are just from a different example manifest? It seems pretty high for a daemon that I assume doesn't need much cpu and memory.

Reply:
I've used similar settings to the rbd-mirror daemon since it does more or less the same job. I believe it's too early to have good numbers.